chapters/scoping.tex

```diff
@@ -99,6 +99,7 @@ \subsection{Composite Name Lookup}\label{composite-name-lookup}
   If the identifier denotes a class, that class is temporarily flattened (as if instantiating a component without modifiers of this class, see \cref{modification-environment}) and using the enclosing classes of the denoted class.
   The rest of the name (e.g., \lstinline!B! or \lstinline!B.C!) is looked up among the declared named elements of the temporary flattened class.
   If the class does not satisfy the requirements for a package, the lookup is restricted to encapsulated\indexinline{encapsulated} elements only.
+  The special case where the class satisfies the requirement for a package, but is not a package, is deprecated.
```

Comment:
Would it improve clarity to use the keyword instead of plain text?  Also repeat the plural form of _requirements_ in the added sentence:
```suggestion
  If the class does not satisfy the requirements for a \lstinline!package!, the lookup is restricted to encapsulated\indexinline{encapsulated} elements only.
  The special case where the class satisfies the requirements for a \lstinline!package!, but is not declared as \lstinline!package!, is deprecated.
```